### PR TITLE
Include .map files in debug image

### PR DESCRIPTION
**Pull request by keithc-ca**
See eclipse/openj9#10099.

### CODE REVIEW

File: closed/DebugImage.gmk [hunk 1]
```diff
@@ -33,7 +33,7 @@ ifeq (true,$(ZIP_DEBUGINFO_FILES))
 else ifeq (macosx,$(OPENJDK_TARGET_OS))
   openj9_debuginfo_filter = $(strip $(foreach path, $1, $(if $(findstring .dSYM/,$(path)),$(path))))
 else ifeq (windows,$(OPENJDK_TARGET_OS))
-  openj9_debuginfo_filter = $(filter %.pdb, $1)
+  openj9_debuginfo_filter = $(filter %.map %.pdb, $1)
```
Comment (pshipton) [inline closed/DebugImage.gmk:36]: Only Windows has .map files?

Comment (keithc-ca) [reply closed/DebugImage.gmk:36]: Yes, I believe so.

**Comment by pshipton** [comment]
jenkins compile win jdk8

**Comment by pshipton** [comment]
jenkins compile win jdk8

**Comment by pshipton** [comment]
Will this affect the builds at Adopt? I'm think we should add it to the 0.21 release.

**Comment by keithc-ca** [comment]
> Will this affect the builds at Adopt? I'm thinking we should add it to the 0.21 release.

Yes: #429 is ready for your consideration.

**Pull request merged**